package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Model-Agnostic Visual Debugging Tool for Machine Learning",
   "license": "Apache-2.0",
+  "private": "true",
```

Comment:
Is this intentional that it's a private module?

Reply:
The root package.json has to be private for the workspace settings, because we are not going to publish the root (we publish each individual modules inside modules/* including manifold), this is fine. Note that deck.gl uses this approach as well.

Reply:
Ah gotcha. In that case, should we update the description and the package name so that it indicates the root package of the monorepo is private rather than the manifold.ml package is private?

Reply:
Hi Chris,

Can we put {"name" : "manifold"} in root package.json to align it with the github repo name for now. Following the inspiration of the deck.gl approach:
```
{
  "name": "deck.gl",
  "description": "A suite of 3D-enabled data visualization overlays, suitable for react-map-gl",
  "license": "MIT",
  "private": "true",
```

Reply:
👍